models/turbine_models/custom_models/stateless_llama.py

```diff
@@ -343,17 +344,19 @@ def run_cached_initialize(
                     HIDDEN_DIM,
                     NUM_LAYERS,
                 )
-                forw_const = (
-                    [x.dynamic_dim(1) < MAX_STEP_SEQ]
-                    + [state_arg[0].dynamic_dim(1) < MAX_STEP_SEQ]
-                    + [
-                        x.dynamic_dim(1) == (state_arg[0].dynamic_dim(1))
-                        for x in state_arg[1:]
-                    ]
-                    + [x.dynamic_dim(1) < MAX_STEP_SEQ for x in state_arg[1:]]
+                state_arg0_dim1 = torch.export.Dim(
+                    "state_arg0_dim1", max=MAX_STEP_SEQ - 1
                 )
+                x_dim = torch.export.Dim("x_dim", max=MAX_STEP_SEQ - 1)
+                dynamic_shapes_forw = {
+                    "arg0_1": {1: x_dim},
+                    "arg1_1": {1: state_arg0_dim1},
+                }
+                for dim_number in range(1, len(state_arg)):
+                    current_dim_dict = {f"arg{dim_number + 1}_1": {1: state_arg0_dim1}}
```

Comment:
really really nit and probably a preference thing, but I think its slightly more readable to make it so dim_number is the actual arg index by adjusting the range arguments.

Reply:
Makes sense, done